package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "awesome-todolist",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "main": "index.js",
+  "scripts": {
```

Comment:
- [ ] Kindly add basic commands for building Webpack in development mode, production mode eg start. This will make it easier to launch your project using npm start.
- [ ] [optional] Also I found this warning below when running npm build, it would be nicer to fix it.
![image](https://user-images.githubusercontent.com/61408860/183941681-bd1a049b-14f0-4690-8903-d4b6c8f7450b.png)

Reply:
- [ ] @shuja-shah great work on implementing the changes requested, kindly ensure to add the start command in your scripts, this will make it possible to run your project using the npm command.